h/static/scripts/test/streamsearch-test.coffee

```diff
@@ -0,0 +1,182 @@
+{module, inject} = require('angular-mock')
+
+assert = chai.assert
+sinon.assert.expose assert, prefix: null
+
+describe 'h:streamsearch', ->
+  before ->
+    angular.module('h', ['ngRoute'])
+    require('../streamsearch')
+
+  describe 'StreamSearchController', ->
```

Comment:
I think this must be my fault... going to look into what happened to the rebase here.